cordapp-example/workflows-kotlin/src/main/kotlin/com/example/utils/getStxStates.kt

```diff
@@ -0,0 +1,16 @@
+package com.example.utils
+
+import net.corda.core.contracts.ContractState
+import net.corda.core.contracts.StateRef
+import net.corda.core.node.ServiceHub
+
+fun Iterable<StateRef>.getStxStates(serviceHub: ServiceHub): List<ContractState> {
+    val states = mutableListOf<ContractState>()
+
+    for (element in this) {
+        val stateAndRef = serviceHub.toStateAndRef<ContractState>(element)
+        states.add(stateAndRef.state.data)
+    }
+
+    return states
```

Comment:
No need to make a `for` loop:

```kotlin
fun Iterable<StateRef>.getStxStates(serviceHub: ServiceHub) = map {
    serviceHub.toStateAndRef<ContractState>(it)
}
```
Not sure you need an extra function for such a simple thing ;)